docs/administration-guide/secure-database-connections-with-ssl-certificates.md

```diff
@@ -0,0 +1,63 @@
+## Securing database connections using an SSL certificate
+
+If you'd like to connect your Metabase Cloud or self-hosted instance to a database, you can secure the connection using Secure Socket Layer (SSL) encryption with a certificate.
+
+Why you'd want to do this:
+
+- You're using Metabase Cloud and want to create a secure connection to a data warehouse that accepts a JDBC connection (e.g., PostgreSQL).
+- You're self-hosting Metabase and want to create a secure connection to an application database hosted by an external provider (this situation is rare).
+
+If you're using Metabase Cloud, the application database is handled for you, so you'd only need to secure connections to data warehouses that you add to your Metabase.
+
+### Prerequisites
+
+A database that allows a JDBC connection, as you'll need to use a connection string to secure the connection.
```

Comment:
you'll need to use a connection string to specify the certificate that you want to use